apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -554,9 +564,13 @@ export default {
 		},
 
 		canChangeHideDownload() {
-			const hasDisabledDownload = (shareAttribute) => shareAttribute.key === 'download' && shareAttribute.scope === 'permissions' && shareAttribute.enabled === false
+			const hasDisabledDownload = (shareAttribute) => shareAttribute.scope === 'permissions' && shareAttribute.key === 'download'&& shareAttribute.value === false
```

Comment:
Maybe keep it readable :sweat_smile:  (or at least add a space between `'download' and '&&') ;)

```suggestion
			const hasDisabledDownload = (shareAttribute) => 
				shareAttribute.scope === 'permissions'
				&& shareAttribute.key === 'download'
				&& shareAttribute.value === false
```